scripts/align-version.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
It would be REALLY nice if we could make all those things `node` scripts instead of `bash`. This would be much friendlier to non-UNIX platforms.

Reply:
This script mainly runs a bunch of shell commands, I feel it's suitable as a shell script.